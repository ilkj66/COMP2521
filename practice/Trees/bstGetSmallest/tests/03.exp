Enter the preorder traversal of the BST: 
BST:

5
 \
  8
 / \
6   9

bstreeGetSmallest returned: 5
