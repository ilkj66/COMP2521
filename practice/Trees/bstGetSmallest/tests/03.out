Enter the preorder traversal of the BST: 
BST:

5
 \
  8
 / \
6   9

bstGetSmallest returned: 5
